src/PKSim.Core/Services/HIDiseaseStateImplementation.cs

```diff
@@ -36,14 +36,14 @@ public static class ChildPughScore
          };
 
       private static readonly HIFactors _portalFlowScalingFactor = createFactors(0.4, 0.36, 0.04);
-      private static readonly HIFactors _hepaticFlowScalingFactor = createFactors(1.3, 2.3, 3.4);
+      private static readonly HIFactors _hepaticFlowScalingFactor = createFactors(1.61, 2.34, 5.29);
       private static readonly HIFactors _hepaticVolumeScalingFactor = createFactors(0.69, 0.55, 0.28);
       private static readonly HIFactors _renalFlowScalingFactor = createFactors(0.88, 0.65, 0.48);
       private static readonly HIFactors _cardiacIndexScalingFactor = createFactors(1.11, 1.27, 1.36);
       private static readonly HIFactors _gfrScalingFactor = createFactors(1, 0.7, 0.36);
       private static readonly HIFactors _albuminScalingFactor = createFactors(0.81, 0.68, 0.5);
       private static readonly HIFactors _agpScalingFactor = createFactors(0.6, 0.56, 0.3);
-      private static readonly HIFactors _hematocritScalingFactor = createFactors(0.92, 0.88, 0.83);
+      private static readonly HIFactors _hematocritScalingFactor = createFactors(0.86, 0.822, 0.778);
```

Comment:
In https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2621#issuecomment-1552358781 HCT-Scalefactor for A is 0.866